src/views/thread/style.js

```diff
@@ -609,6 +609,13 @@ export const WatercoolerActionBarContainer = styled(ActionBarContainer)`
   margin-bottom: 16px;
 `;
 
+export const FixedBottomActionBarContainer = styled(ActionBarContainer)`
+  z-index: 1;
+  bottom: 0;
+  position: sticky;
```

Comment:
What is the fallback behaviour if a browser doesn't support `sticky`, will anything break?

Reply:
Double checked this in IE11 and the behaviour remains the same as before this change :+1: